builtin/game/item_entity.lua

```diff
@@ -58,6 +58,8 @@ core.register_entity(":__builtin:item", {
 		local count = math.min(stack:get_count(), max_count)
 		local size = 0.2 + 0.1 * (count / max_count) ^ (1 / 3)
 		local coll_height = size * 0.75
+		local def = minetest.registered_items[itemname]
```

Comment:
Noticed this right now:
`light_source` is only valid for nodes, so applying it for items wouldn't be exactly what the documentation states (somewhen there will be a new issue about this inconsistency).
Also, builtin uses the internal namespace/table name -> `core.registered_nodes`